### PR TITLE
CHAP-2266 - Order Status Card

### PR DESCRIPTION
**Pull request by nkiefera9s**

### CODE REVIEW

File: src/templates/cards/OrderStatusCard/OrderItem.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+              variant='label2'
+              color='text.secondary'
+              className='label'
+            >
+              Versendet am:
+            </Typography>
+            <Typography variant='button2'>{sendAt}</Typography>
+          </Box>
+        )}
+        {onClick && <Button onClick={onClick}>Lieferung verfolgen</Button>}
```
Comment (Momen-a9s) [inline src/templates/cards/OrderStatusCard/OrderItem.tsx:81]: not sure how we can handle the loocalizatin

File: src/templates/cards/OrderStatusCard/OrderItem.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+export const OrderItem: React.FC<OrderItemProps> = (props) => {
+  const {
+    title,
+    image,
+    backgroundImage = false,
+    variant,
+    amount,
+    points,
+    sendAt,
+    onClick
```
Comment (Momen-a9s) [inline src/templates/cards/OrderStatusCard/OrderItem.tsx:26]: onClick was for tracking the order or click on the order to redirect to Order details

**Review by Momen-a9s — COMMENTED** [review]
Thanks great work, and looks great in storybook viewer 👍 

do we need to import/export the component to the index ?

File: src/templates/cards/OrderStatusCard/OrderItem.tsx (hunk 1 above)
Comment (nkiefera9s) [reply src/templates/cards/OrderStatusCard/OrderItem.tsx:81]: Maybe via props 🤔

File: src/templates/cards/OrderStatusCard/OrderItem.tsx (hunk 2 above)
Comment (nkiefera9s) [reply src/templates/cards/OrderStatusCard/OrderItem.tsx:26]: This one is for tracking the order :D So the onClick handler on the button inside the orderItem

**Pull request merged**